Library/Homebrew/build.rb

```diff
@@ -132,6 +132,8 @@ def install
       else
         formula.prefix.mkpath
 
+        (f.logs/"00.options.out").write \
+          "#{f.full_name} #{@formula.build.used_options.join(" ")}"
```

Comment:
`f` => `formula`. Also, `Options` doesn't have the `join` method, but it includes `Enumerable`, so we can use `sort`. Putting it together:

```ruby
        (formula.logs/"00.options.out").write \
          "#{formula.full_name} #{@formula.build.used_options.sort.join(" ")}"
```

Example gist logs with this patch: https://gist.github.com/8a025e9d9b0bedaf7a761440b87b06dd.